SchedulePlatform.Api/Controllers/MenuController.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using SchedulePlatform.Api.Mappings;
+using SchedulePlatform.Api.Models;
+using SchedulePlatform.Models.Entities;
+using SchedulePlatform.Service.Interfaces;
+
+namespace SchedulePlatform.Api.Controllers
+{
+	[Route("Api/[Controller]")]
+
+	[ApiController]
+
```

Comment:
I added some comments in a previous PR about spacing, indentation and useless new lines - they all apply on this PR as well.